client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -140,13 +140,18 @@ export const ChapterPage: NextPageWithLayout = () => {
             </HStack>
           )}
           {checkPermission(user, Permission.UsersView, { chapterId }) && (
-            <Box>
-              <LinkButton href={`${chapterId}/users`} paddingBlock={'2'}>
-                Chapter Users
-              </LinkButton>
-            </Box>
+            <LinkButton
+              href={`${chapterId}/users`}
+              paddingBlock="2"
+              marginBlock="1.5em"
+            >
+              Chapter Users
+            </LinkButton>
           )}
-          <HStack mt={'2'}>
+          <Grid
+            gridTemplateColumns="repeat(auto-fit, minmax(6.5rem, 1fr))"
```

Comment:
With the non-mobile width buttons get stretched a bit much...
![image](https://user-images.githubusercontent.com/60067306/211406468-5e3d050f-4678-4cb3-8501-2c4dfefacdc1.png)


Changing it to `auto-fill` keeps some maximum button width, it might be less flexible
```suggestion
            gridTemplateColumns="repeat(auto-fill, minmax(6.5rem, 1fr))"
```
![image](https://user-images.githubusercontent.com/60067306/211406632-852b55c6-7fda-4a5a-ad15-bf8bf65e79b2.png)

What do you think? Maybe you have other ideas?

Reply:
Nope, I copied my previous code without deep thoughts. Good catch 👍